js/ai.js

```diff
@@ -68,10 +68,10 @@ export default {
         this.foundSet = false;
 
         //have ai check each time it resumes if game should end; copied from board logic
```

Comment:
nit: add space at start of comment.
nit: no longer true that it was copied from board logic.

Reply:
Updated; now reads

// have ai check each time it resumes if game should end; this logic used to be in board.js before card draw